fabtests/functional/rdm_tagged_peek.c

```diff
@@ -282,15 +263,15 @@ static int do_sends(void)
 	(void) ft_fill_buf(tx_buf, tx_size);
 
 	for (i = 1; i <= SEND_CNT; i++) {
-		ret = fi_tsend(ep, tx_buf, tx_size, mr_desc,
-				remote_fi_addr, BASE_TAG + i,
-				&tx_ctx_arr[i].context);
+		ret = ft_post_tx_buf(ep, remote_fi_addr, tx_size, NO_CQ_DATA, &tx_ctx_arr[i].context, tx_buf, mr_desc, BASE_TAG + i);
```

Comment:
ugh... please don't make me scroll left and right

Reply:
Fixed.